pipeline/http_post.go

```
@@ -14,13 +14,48 @@ package pipeline
 type HTTPPostNode struct {
 	chainnode
 
+	HTTPPostHandlers []*HTTPPostHandler `tick:"Endpoint"`
+
 	// tick:ignore
 	Url string
 }
 
-func newHTTPPostNode(wants EdgeType, url string) *HTTPPostNode {
+func newHTTPPostNode(wants EdgeType, urls ...string) *HTTPPostNode {
+	var url string
+	// TODO: What to do if len(urls) > 1?
+	if len(urls) != 0 {
+		url = urls[0]
```

We should add a validate method to this node and we can return the error there.